src/Castle.Core.Tests/Castle.Core.Tests.csproj

```diff
@@ -17,6 +17,7 @@
 		<AssemblyOriginatorKeyFile>..\..\buildscripts\CastleKey.snk</AssemblyOriginatorKeyFile>
 		<PublicSign Condition="'$(OS)'=='Unix'">true</PublicSign>
 		<StartupObject>Program</StartupObject>
+		<InvariantGlobalization>true</InvariantGlobalization>
```

Comment:
As above (see comment in `Castle.Core.Tests.WeakNamed.csproj`), please undo this change if it isn't strictly required for offloading `DiagnosticsLogger` into a separate project/package.